billing.rst

```diff
@@ -0,0 +1,146 @@
+=========== =================================================== ======== ========== ======
+Reference   Designation                                         Quantity Unit price Total
+=========== =================================================== ======== ========== ======
+SC_DEV       | Unused time on 1 × SymfonyCloud Development plan prorated  €10.00    -€0.70
+             | From July 31, 2020 to August 2, 2020
+SC_STANDARD  | Remaining time on 1 × SymfonyCloud Standard plan prorated  €50.00     €3.49
+             | From July 31, 2020 to August 2, 2020
+SC_STANDARD  | 1 × SymfonyCloud Standard plan
+             | From August 2, 2020 to September 2, 2020         1         €50.00    €50.00
+----------- --------------------------------------------------- -------- ---------- ------
+\           **Total before taxes**                                                  €52.79
+=========== =================================================== ======== ========== ======
+
+Am I gonna be charged after trial?
+----------------------------------
+
+SymfonyCloud billing is opt-in based. You will be billed only if you explicitely
+want to keep your project around after trial and accept billing using ``symfony
+project:billing:accept`` or if you move to a Production plan using ``symfony
+project:scale``.
+
+If you do not upscale your project or accept billing, you project will be
+automatically suspended at the end of your trial. If you do no pay in the days
+following its suspension, your project will be automatically deleted.
+
+What is the difference between development and production plans?
+----------------------------------------------------------------
+
+The development plan is intended to let you test and prepare your project before
+it going live. You do not have dedicated resources and can not attach production
+domains to your Main environment.
+
+When a project is attached to a production plan, the Main environment runs on
+different hosts with dedicated ressources and can serve traffic to custom
+production domain names.
+
+How do I change my project plan?
+--------------------------------
+
+To upscale your project one step up (from Standard to Medium for example), use
+``symfony project:scale``. To downscale one step down, use ``symfony
+project:scale --down``.
+
+You can also scale directly to a specific plan using ``symfony project:scale
+<plan>`` (replace ``plan`` with the plan name).
+
+How do I add more storage to my project?
+----------------------------------------
+
+Your subscription comes with 5GB of persistent storage per environment. Adding
+more storage is made per increment of 5GB and is billed $3.60 / 3,00€ per
+increment and per environment.
+
+Add more storage using ``symfony project:edit storage X`` (replacing ``X`` by
+the desired storage amount).
+
+How do I add more environments to my project?
+---------------------------------------------
+
+Your subscription comes with 3 non-production and active environments. You can
```

Comment:
Is it 3 non-prod envs or 1 prod env + 2 non-prod? In any case, we should mention the prod env.